test/functional/wallet_listtransactions.py

```diff
@@ -284,6 +285,17 @@ def run_invalid_parameters_test(self):
         assert_raises_rpc_error(-8, "Negative count", self.nodes[0].listtransactions, count=-1)
         assert_raises_rpc_error(-8, "Negative from", self.nodes[0].listtransactions, skip=-1)
 
+    def test_op_return(self):
+        """Test if OP_RETURN outputs will be displayed correctly."""
+        raw_tx = self.nodes[0].createrawtransaction([],[{'data':'aa'}])
```

Comment:
In your diff in both test files, if you retouch
```suggestion
        raw_tx = self.nodes[0].createrawtransaction([], [{'data': 'aa'}])
```

Reply:
Done in 078199c245cc475bb594930177fd505d84fede4b. Thanks.

Reply:
Also in https://github.com/bitcoin/bitcoin/pull/26349/files#diff-33258976a327dfd9623df9379f0fd15e15c32a1488d658912e3973d2e0bf7a74R456 if you retouch (thanks!)

Reply:
Done in https://github.com/bitcoin/bitcoin/pull/26349/commits/b725cbea13b0f4774abbdf939b3ffe224fc63696. Thanks.